Configuration/ProcessModifiers/python/approximatedClusters_cff.py

```diff
@@ -0,0 +1,3 @@
+import FWCore.ParameterSet.Config as cms
+
+approximatedClusters =  cms.Modifier()
```

Comment:
I am wondering if the name of the modifier should expose the fact that these are SiStrip clusters.

Reply:
Good point.  How about approxSiStripClusters?